resources/measures/HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -1113,28 +1113,26 @@ def self.process_ducts_for_unit(model, runner, ducts, building, unit, unit_index
 
     location_frac_conduction = location_frac_leakage
     ducts.num_returns = Airflow.get_num_returns(ducts.num_returns, num_stories)
-    supply_surface_area = Airflow.get_duct_supply_surface_area(ducts.supply_area_mult, unit_ffa, num_stories)
-    return_surface_area = Airflow.get_return_surface_area(ducts.return_area_mult, unit_ffa, num_stories, ducts.num_returns)
+    supply_surface_area = Airflow.get_duct_supply_surface_area(unit_ffa) * ducts.supply_area_mult
+    return_surface_area = Airflow.get_duct_return_surface_area(unit_ffa, ducts.num_returns) * ducts.return_area_mult
 
     # Calculate Duct UA value
     if location_name != unit_living.zone.name.to_s
-      unconditioned_duct_area = supply_surface_area * location_frac_conduction
       supply_r = Airflow.get_duct_insulation_rvalue(ducts.r, true)
       return_r = Airflow.get_duct_insulation_rvalue(ducts.r, false)
-      unconditioned_ua = unconditioned_duct_area / supply_r
+      supply_ua = supply_surface_area / supply_r
       return_ua = return_surface_area / return_r
```

Comment:
It looks like this change means that neither supply nor return ua would use the location_frac. Instead I think we want both of them to use location_frac.